ui/tests/integration/components/sync/sync-header-test.js

```diff
@@ -28,7 +29,7 @@ module('Integration | Component | sync | SyncHeader', function (hooks) {
     );
 
     assert.dom(PAGE.title).hasText('Secrets sync Enterprise feature');
-    assert.dom(PAGE.breadcrumbs).doesNotExist('does not render breadcrumbs');
+    assert.dom(PAGE.headerContainer).hasTextContaining('Secrets sync', 'renders default breadcrumb');
```

Comment:
I'm not sure why this wasn't failing before. Maybe it was and we just didn't catch it but the default breadcrumb renders regardless of the version:
```hbs
<Page::Breadcrumbs @breadcrumbs={{or @breadcrumbs (array (hash label="Secrets sync"))}} />
```

Reply:
If the header container includes the title is this passing due to that and not breadcrumbs? Seems like we might want a specific breadcrumbs selector

Reply:
That's a good question. I didn't set up these selectors and just copied the assertion from another test that was checking for breadcrumbs but I agree that checking that the header contains the text is not as pointed as actually checking for the breadcrumb specifically. Let me update that.